src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/CommandItemViewModel.cs

```diff
@@ -80,11 +80,11 @@ public override void InitializeProperties()
         Title = model.Title;
         Subtitle = model.Subtitle;
         IconUri = model.Icon.Icon;
-        MoreCommands = model.MoreCommands
+        /*MoreCommands = model.MoreCommands
```

Comment:
Oh right @zadjii-msft this was causing issues for me with a System.ArgumentNullException Parameter 'source'

I'll try and remember to uncomment and test once I get everything else running and now that I've pulled some updates. Just curious if you'd seen anything like that before.

Reply:
I just realized hitting another suspiciously similar looking issue elsewhere, that I had an old extension still installed, so it was trying to load that, even though we've changed a ton of things... Seems like we need a version check/schema maybe for these scenarios?